### DISCUSSION TITLE
"Error: "StatusBar" plugin is not implemented on android"

### DISCUSSION THREAD

**Discussion by andreashellstrom** [quasarframework/quasar#9413] [CLI - Capacitor / Cordova modes] [answered]
I cannot get the @capacitor/status-bar plugin to work. I keep getting this error  in Android Studio emulator.

I have these installed:

![image](https://user-images.githubusercontent.com/84782609/119499262-d99a1600-bd66-11eb-93bb-bb6e9144d6f9.png)

In my App.vue I init statusbar as this:

`import { Vue, Component, Watch } from "vue-property-decorator";
import { setting } from "src/common/settings";
import { customerService } from "./services/customer.service";
import { StatusBar } from "@capacitor/status-bar";

@Component
export default class App extends Vue {
    async mounted() {
        try {
            await StatusBar.setBackgroundColor({ color: "#d69300" });
        } catch (error) {
            alert("Statusbar error: " + error);
        }
    }
}`

The command 'quasar info' shows this:

![image](https://user-images.githubusercontent.com/84782609/119499771-54fbc780-bd67-11eb-9a7b-ea7c7bed7127.png)

Anyone can help?

**Comment by rstoenescu** [comment, marked as answer]
Have you installed `@capacitor/status-bar` in `/src-capacitor`? (`yarn add @capacitor/status-bar` or npm install ....)

**Reply by andreashellstrom** [reply]
I believe this was the solution, thanks!

**Comment by ontwikkelfabriek** [comment]
Looking at your `qusar info` output, some other things that I think are wrong for Quasar V1 app:

- NodeJS should be `v12.x` instead of `14.x`
- `yarn` should be used for `local` deps, `npm` for `global` deps

**Comment by andreashellstrom** [comment]
I got it working now! I'm not entirely 100% sure what the solution was, I tried a lot of things. But I believe it started working when I installed @capacitor/status-bar in the src-capacitor folder (instead of the ClientApp-folder). Just as rstoenescu suggested. 

After "quasar dev -m capacitor -T android", this is how my Android Studio project looks like:

![image](https://user-images.githubusercontent.com/84782609/119619128-f20c3e00-be03-11eb-9cce-c3027005373e.png)

As you can see, it has built a separate project with the StatusBar plugin java-files. I didn't get any java-files at all before, so that was why it didn't work.

So just to cofirm, should plugins be installed in src-capacitor folder?

**Reply by rstoenescu** [reply]
Capacitor plugins > Yes, always install in src-capacitor
